hooks/tk-multi-loader2/basic/scene_actions.py

```diff
@@ -113,6 +114,23 @@ def generate_actions(self, sg_publish_data, actions, ui_area):
                 }
             )
 
+        if "import_with_options" in actions:
+            vred_version = int(self.parent.engine.vred_version.replace(".", ""))
+            if vred_version >= 202210:
```

Comment:
This will fail if `vred_version=2022.2` since 2022.2 => 20222, which is less than 202210, but 2020.2 should be greater than 2022.1.0

I would suggest using the same function that was added to `tk-alias` for comparing versions: https://github.com/shotgunsoftware/tk-alias/blob/8d8fd2f53d41bd999431092640beb810dc60ba74/python/tk_alias/menu_generation.py#L191-L228

We could add this function to `engine.py`, and it can also be used to replace the comparison logic on lines `93` and `103`.

Reply:
Great suggestion, I have introduced it and changed the comparisons in the engine and the hook.